app/models/budget/investment/milestone.rb

```diff
@@ -8,7 +8,7 @@ class Milestone < ActiveRecord::Base
                    accepted_content_types: [ "application/pdf" ]
 
       translates :title, :description, touch: true
-      globalize_accessors locales: [:en, :es, :fr, :nl, :val, :pt_br]
+      globalize_accessors locales: I18n.available_locales.map { |l| l.to_s.underscore.to_sym }
```

Comment:
No, I think you're right. I actually introduced a bug implementing this same logic here by neglecting to turn the locale back into a symbol. That wouldn't have happened if we had the logic in one place.

Right now we have the logic in views, models, and `db/dev_seeds/`. I guess the Rails solution for this is to create a `LocaleNamesHelper` or something in `lib/` to put this method in. I can't think of what else we would put in a class like that though. Another alternative would be to just use `GlobalizeHelper.neutral_locale()` from the model and seed files.

What do you think?

Reply:
Adding something to `lib/` sounds good to me. I'd avoid ending the class with `Helper`, though, since the name suggests a module only available in the views.

Reply:
Actually I'm not convinced this whole `neutral_locale` thing is necessary, or at least to apply it in so many places. The original PR says:

>There was a problem with the portuguese locale. It was pt-BR, but globalize_accessors gem doesn't allow the creation of methods using locales with that format (with a -, like es-ES, en-GB, etc.).
> 
> To avoid transforming the locale pt-BR to pt, losing the distinction of the different variations of the language, a function has been added to transform pt-BR into pt_br (without changing the locale itself). That way, when globalize uses the locales, all of them will have a valid and consistent format (downcased and uderscored). 

It seems the only issue is Ruby doesn't allow `-` in method names. I feel like that could be handled by the gem, or even externally but at a different level... I'll look into it later.

Reply:
Not even necessary, as the problem is completely avoided by  https://github.com/AyuntamientoMadrid/consul/pull/1613/commits/49c1cdf5b6e1510b45afcc0a9cdd24c4cc6ddce6 :raised_hands: